src/Components/SmartComponents/SystemsExposedTable/SystemsExposedTable.js

```diff
@@ -37,11 +38,12 @@ const SystemsExposedTable = (props) => {
     const [selectedHosts, setSelectedHosts] = useState(undefined);
     const inventory = React.createRef();
     const dispatch = useDispatch();
-
+    const createUrlParams = useCreateUrlParams(props.history);
     const affectedSystems = useSelector(({ CVEStore }) => CVEStore.affectedSystemsByCVE);
+    const urlParams = qs.parse(props.location.search);
```

Comment:
instead of grabbing the `urlParams` here why don't you return in you custom hook
the `useCreateUrlParams` will return an array 

     const [createUrlParams, urlParams]   = useCreateUrlParams(props.history);

Reply:
Great idea, I will adopt it.